src/main/java/net/io/kino/service/impl/ReservationServiceImpl.java

```diff
@@ -17,21 +21,32 @@
     @Autowired
     private TicketTypesRepository ticketTypes;
 
+    @Autowired
+    private EmailSender emailSender;
+
+    private TicketService ticketService = new TicketServiceImpl();
+
+//todo czy bilet ma byc zapiswany w bazie przy tworzeniu zamówienia czy przy potwierdzeniu zamówienia
     @Override
     public boolean createOrder(List<Ticket> tickets, PersonalDetails client) {
-        Order order = new Order(tickets,client, OrderState.wTrakcieRealizacji);
+        Order order = new Order(tickets,client, OrderState.inProgress);
         return orders.save(order)!=null;
     }
 
     @Override
     public boolean confirmOrder(Order order) {
-        order.setState(OrderState.oplacone);
+        order.setState(OrderState.paid);
+        for (Ticket ticket:order.getTickets()) {
+            ticketService.confirmTicket(ticket);
+        }
+        emailSender.sendEmail(order.getClient().getEmail(),order);
         return orders.save(order)!=null;
     }
 
     @Override
     public boolean cancelOrder(Order order) {
-        order.setState(OrderState.anulowane);
+        order.setState(OrderState.cancelled);
+        order.getTickets().clear();
```

Comment:
Myślę nad tym, czy to po pierwsze się powinno czyścić
a po drugie to jak będzie sprawdzane np. ile osób jest na seans
w sensie czy tutaj powinien być jakoś bilet deaktywowany
czy będą pobieranie bilety z warunkiem, że order do którego należy nie jest deaktywowany

Reply:
W sumie to wyżej w confirmOrder jest aktywacja każdego z biletów, to mogłaby tutaj być deaktywacja.

Jakieś rzeczy typu pobieranie biletów na dany film to kolejna integracja będzie

Reply:
Wydaje mi się, że sam stan Order wystarczy, bez aktywacji biletów. Jak będziemy chcieli pobierać bilety na seans, to po prostu weźmiemy wszystkie nieanulowane Ordery. Ticket nie musi się tym przejmować.